### DISCUSSION TITLE
Can't generate order

### DISCUSSION THREAD

**Discussion by wei-hao-ming** [opencart/opencart#12523] [Q&A]
![H4 B9K8O0Z8ZEL8SRUNG%HV](https://github.com/opencart/opencart/assets/74048274/4c3769e8-ab29-49ce-9445-e93f87221876)
What is causing this and is there a workaround?

**Comment by felipoantonoff** [comment]
Hard to know what causes it with just this image.

Error 502 can be several things, it would be necessary to discover the real error and thus analyze how to solve it.
